CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
 # Changelog
 
+### 7.0.0
+
+* __(breaking change)__ RSpec in Queue Mode:
+  * The default for `KNAPSACK_PRO_LOG_LEVEL` is `info` instead of `debug`.
+  * The RSpec `before(:suite)` and `after(:suite)` hooks change:
+
+    __Before:__<br>
+    The `before(:suite)` and `after(:suite)` hooks are executed multiple times. Each time for a set of tests fetched from Knapsack Pro Queue API.
+
+    __After:__<br>
+    The `before(:suite)` and `after(:suite)` hooks are executed only once. The `before(:suite)` hook is executed before starting tests. The `after(:suite)` hook is executed after all tests are completed. It is the default RSpec behavior expected by users.
+
+  * The `KnapsackPro::Hooks::Queue.after_queue` hook change:
+
+    __Before:__<br>
+    The `KnapsackPro::Hooks::Queue.after_queue` hook is executed outside of the `after(:suite)` hook.
+
+    __After:__<br>
+    The `KnapsackPro::Hooks::Queue.after_queue` hook is executed __inside__ of the `after(:suite)` hook.
+
+* RSpec recommended changes in your project:
+  * Remove the following code if you use Queue Mode and the `rspec_junit_formatter` gem to generate JUnit XML or JSON reports. The `KnapsackPro::Hooks::Queue.after_subset_queue` hook is no longer needed to generate a valid XML/JSON report.
+
+    ```ruby
+    # REMOVE THE FOLLOWING CODE
+    # spec_helper.rb or rails_helper.rb
+    TMP_REPORT = "tmp/rspec_#{ENV['KNAPSACK_PRO_CI_NODE_INDEX']}.xml"
+    FINAL_REPORT = "tmp/final_rspec_#{ENV['KNAPSACK_PRO_CI_NODE_INDEX']}.xml"
+
+    KnapsackPro::Hooks::Queue.after_subset_queue do |queue_id, subset_queue_id|
+      if File.exist?(TMP_REPORT)
+        FileUtils.mv(TMP_REPORT, FINAL_REPORT)
+      end
+    end
+    ```
+
+    You can keep the Knapsack Pro command unchanged with the `--format RspecJunitFormatter --out tmp/rspec.xml` options.
+
+    ```bash
+    bundle exec rake "knapsack_pro:queue:rspec[--format documentation --format RspecJunitFormatter --out tmp/rspec.xml]"
+    ```
+
+    Reference in [docs](https://docs.knapsackpro.com/ruby/rspec/#formatters-rspec_junit_formatter-json).
+
+    See [docs](https://docs.knapsackpro.com/ruby/circleci/) if you use CircleCI.
+
+  * Replace the following if you are using Queue Mode and have a version of the `percy-capybara` gem that is older than version `4`:
+
+    Before:
+
+    ```ruby
+    KnapsackPro::Hooks::Queue.before_queue { |queue_id| Percy::Capybara.initialize_build }
+    KnapsackPro::Hooks::Queue.after_queue { |queue_id| Percy::Capybara.finalize_build }
+    ```
+
+    After:
+
+    ```ruby
+    # recommended
+    before(:suite) { Percy::Capybara.initialize_build }
+    after(:suite) { Percy::Capybara.finalize_build }
+    ```
+
+    Reference in [docs](https://docs.knapsackpro.com/ruby/hooks/#percy-capybara) for the latest Percy version.
```

Comment:
```suggestion
    Learn more about [using Knapsack Pro with Percy](https://docs.knapsackpro.com/ruby/rspec/#formatters-rspec_junit_formatter-json) in the docs.
```

Reply:
done: https://github.com/KnapsackPro/knapsack_pro-ruby/pull/237/commits/5341b897b0c7c0bb6fceec63419ec2a84cdc9459 I fixed the wrong url.